fission/.eslintrc.cjs

```diff
@@ -15,13 +23,103 @@ module.exports = {
       { allowConstantExport: true },
     ],
     '@typescript-eslint/no-unused-vars': [
-      'warn', // or "error"
+      'warn',
+      {
+        argsIgnorePattern: '^_',
+        varsIgnorePattern: '^_',
+        caughtErrorsIgnorePattern: '^_',
+      },
+    ],
+    '@typescript-eslint/naming-convention': [
+      'warn',
+      {
+        selector: ['variable', 'parameter'],
+        modifiers: ['unused'],
+        format: ['camelCase'],
+        leadingUnderscore: 'require',
+        trailingUnderscore: 'forbid',
+      },
+      {
+        selector: ['function', 'method', 'classMethod'],
+        format: ['camelCase'],
+        leadingUnderscore: 'allow',
+        trailingUnderscore: 'forbid',
+      },
+      {
+        selector: ['memberLike'],
+        modifiers: ['private'],
+        format: ['camelCase'],
+        leadingUnderscore: 'require',
+        trailingUnderscore: 'forbid',
+      },
+      {
+        selector: ['variable'],
+        modifiers: ['const'],
+        format: ['PascalCase', 'camelCase', 'UPPER_CASE'],
+      },
+      {
+        selector: ['variableLike'],
+        format: ['camelCase', 'UPPER_CASE'],
+        leadingUnderscore: 'forbid',
+        trailingUnderscore: 'forbid',
+      },
```

Comment:
`variableLike` is sugar for `function` and `parameter` and `variable`, so I believe this rule is either redundant or will conflict with other rules.

https://typescript-eslint.io/rules/naming-convention/